cpp/src/examples/compute_example.cpp

```diff
@@ -49,42 +50,69 @@ int main() {
   status = CreateTable(ctx, rows, &table);
 
   if ((status.is_ok() && table->Columns() == 2 && table->Rows() == rows)) {
-    table->Print(0, 2, 0, 2);
+    table->Print();
   } else {
     std::cout << "table creation failed! " << status.get_msg() << std::endl;
     return 1;
   }
 
-  if ((status = cylon::compute::Sum(table, 1, &result)).is_ok()) {
+  if ((status = cylon::compute::Sum(table, agg_index, &result)).is_ok()) {
     const std::shared_ptr<arrow::DoubleScalar>
         &aa = std::static_pointer_cast<arrow::DoubleScalar>(result->GetResult().scalar());
     std::cout << "sum " << aa->value << " " << status.get_code() << std::endl;
   } else {
     std::cout << "sum failed! " << status.get_msg() << std::endl;
   }
 
-  if ((status = cylon::compute::Count(table, 1, &result)).is_ok()) {
-    const auto &aa = std::static_pointer_cast<arrow::Int64Scalar>(result->GetResult().scalar());
+  if ((status = cylon::compute::Count(table, agg_index, &result)).is_ok()) {
+    const auto &aa = std::static_pointer_cast<arrow::DoubleScalar>(result->GetResult().scalar());
     std::cout << "count " << aa->value << " " << status.get_code() << std::endl;
   } else {
     std::cout << "count failed! " << status.get_msg() << std::endl;
   }
 
-  if ((status = cylon::compute::Min(table, 1, &result)).is_ok()) {
+  if ((status = cylon::compute::Min(table, agg_index, &result)).is_ok()) {
     const auto &aa = std::static_pointer_cast<arrow::DoubleScalar>(result->GetResult().scalar());
     std::cout << "min " << aa->value << " " << status.get_code() << std::endl;
   } else {
     std::cout << "min failed! " << status.get_msg() << std::endl;
   }
 
-  if ((status = cylon::compute::Max(table, 1, &result)).is_ok()) {
+  if ((status = cylon::compute::Max(table, agg_index, &result)).is_ok()) {
     const std::shared_ptr<arrow::DoubleScalar>
         &aa = std::static_pointer_cast<arrow::DoubleScalar>(result->GetResult().scalar());
     std::cout << "max " << aa->value << " " << status.get_code() << std::endl;
   } else {
     std::cout << "max failed! " << status.get_msg() << std::endl;
   }
 
+  // Adding Python funcs
+  std::shared_ptr<cylon::Table> output;
```

Comment:
Let's add these parts to the tests as well. 
https://github.com/cylondata/cylon/blob/master/cpp/test/aggregate_test.cpp